### DISCUSSION TITLE
Gitoxide: A peek into the future

### DISCUSSION THREAD

**Discussion by mmwinther** [gitpython-developers/GitPython#1637] [Q&A] [answered]
Gitoxide looks like a very promising project, and there's a clear advertisement for it on the README of this repo. Does that mean that Gitoxide is seen as a potential replacement for Gitpython?

My concrete question is: Will Gitoxide offer Python bindings ala Polars which is available for both Rust and Python?

**Comment by Byron** [comment, marked as answer]
I think it will be an alternative instead of a replacement. Those using GitPython will want to keep its API, and `gitoxide` has it's own API which will be made available from Python. Those who want to write reliable software will have no choice but to port to the `gitoxide` bindings for python, but at least then there will be a way out.
